package.json

```diff
@@ -24,8 +24,10 @@
 		"README.md"
 	],
 	"scripts": {
-		"build": "tsc",
-		"build:release": "tsup src/index.ts --format cjs --outDir lib --dts",
+		"build": "pnpm build:clean && pnpm build:esm && pnpm build:cjs",
```

Comment:
`tsup` can output [two module formats at the same time](https://tsup.egoist.dev/#bundle-formats):

```shell
tsup src/index.ts --format cjs,esm
```

How about simplifying this into just `build: tsup src/index.ts --format cjs,esm --outdir lib --dts`?